modules/porous_flow/src/bcs/PorousFlowEnthalpySink.C

```diff
@@ -0,0 +1,90 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "PorousFlowEnthalpySink.h"
+#include "Function.h"
+#include "PorousFlowDictator.h"
+#include "SinglePhaseFluidProperties.h"
+#include "MooseVariable.h"
+#include "PorousFlowSinkBC.h"
+
+#include "libmesh/quadrature.h"
+
+#include <iostream>
+
+registerMooseObject("PorousFlowApp", PorousFlowEnthalpySink);
+
+InputParameters
+PorousFlowEnthalpySink::validParams()
+{
+  InputParameters params = IntegratedBC::validParams();
+  params += PorousFlowSinkBC::validParamsCommon();
+  params.addRequiredParam<UserObjectName>(
+      "PorousFlowDictator", "The UserObject that holds the list of PorousFlow variable names");
+  params.addParam<unsigned int>(
+      "fluid_phase",
+      0,
+      "If supplied, then this BC will potentially be a function of fluid pressure.");
```

Comment:
This is a strange doco string.  This BC is always a function of fluid pressure.  Perhaps something like "Evaluate enthalpy at the pressure of this fluid phase"

Reply:
I think this might be cause by me doing a copy-and-paste...